pkgs/applications/graphics/sane/backends/default.nix

```diff
@@ -62,6 +78,10 @@ stdenv.mkDerivation {
     ++ lib.optional (libusb1 != null) "--with-usb"
   ;
 
+  # autoconf check for HAVE_MMAP is never set on cross compilation.
+  # The pieusb backend fails compilation if HAVE_MMAP is not set.
+  buildFlags = lib.optionals (stdenv.hostPlatform != stdenv.buildPlatform) [ "CFLAGS=-DHAVE_MMAP=1" ];
```

Comment:
We can't use canExecute here?

Reply:
no because its because of cross-compilation not because something is being executed